docs/Fields.md

```diff
@@ -610,7 +610,7 @@ You can find components for react-admin in third-party repositories.
 
 ## TypeScript
 
-All field components accept a generic type that describes the record. This lets TypeScript validate that the `source` prop targets an actual field of the record:
+All field components accept a generic type that describes the record. This lets TypeScript validates that the `source` prop targets an actual field of the record:
```

Comment:
Nope, grammar was good on this one, 'lets' is already conjugated :grin:

Reply:
my bad